packages/pds/src/db/util.ts

```diff
@@ -1,22 +1,15 @@
 import {
   DummyDriver,
   DynamicModule,
+  Kysely,
   RawBuilder,
   SelectQueryBuilder,
   sql,
   SqliteAdapter,
   SqliteIntrospector,
   SqliteQueryCompiler,
 } from 'kysely'
-import DatabaseSchema from './database-schema'
-
-export const actorWhereClause = (actor: string) => {
-  if (actor.startsWith('did:')) {
-    return sql<0 | 1>`"did_handle"."did" = ${actor}`
-  } else {
-    return sql<0 | 1>`"did_handle"."handle" = ${actor}`
-  }
-}
+import { DynamicReferenceBuilder } from 'kysely/dist/cjs/dynamic/dynamic-reference-builder'
```

Comment:
I wonder if you can get away with `ReferenceExpression`, which I think is public.

Reply:
switched it out & it seems to work fine 👌